src/components/ResourceCard/AddResource/ResourceActivityType.js

```diff
@@ -43,7 +45,7 @@ let ResourceActivityType = (props) => {
           <div
             className="activity-img"
             style={{
-              backgroundImage: `url(${global.config.resourceUrl}${activityType.image})`,
+              backgroundImage: `url(${activityType.image})`,
```

Comment:
Do we save full url for activity type image?
In the database, I saw that it is from API storage path

Reply:
full url will work